src/Core/System/SalesChannel/Aggregate/SalesChannelType/SalesChannelTypeEntity.php

```diff
@@ -118,7 +76,7 @@ public function setDescriptionLong(?string $descriptionLong): void
         $this->descriptionLong = $descriptionLong;
     }
 
-    public function getCoverUrl(): string
+    public function getCoverUrl(): ?string
```

Comment:
Is it okay to make these changes?

Reply:
@MartinKrzykawski did it already better :grin: